src/interface.hpp

```diff
@@ -238,30 +227,22 @@ class Interface
         // Start Blinking Cursor for Active connection session.
         new_session->m_sequence_parser->setCursorActive(true);
 
-        // Start Async Connection
-        boost::asio::async_connect(new_connection->socket(),
-                                   endpoint_iterator,
-                                   [this, new_connection, system_connection, new_session]
-                                   (boost::system::error_code ec, tcp::resolver::iterator)
+        // Setup the Connection String
+        std::string connection_string = system_connection->ip;
```

Comment:
Note: Connection String should be switch to Vector<String> so were not splitting on colon on the other side.  This leaves room for error if a password or a colon in it.. for some reason.. :)